broadinstitute/firecloud-orchestration
# AJ-2026 render local config with GSM instead of vault

https://broadworkbench.atlassian.net/browse/AJ-2026

Replaces the vault-dependent local-config render script with one that uses GSM.  
To test:
1. Check out a new repo to avoid previously-rendered configs.
2. Make sure you are on Broad internet or VPN.
3. Run `./local-dev/bin/render`
4. Make sure `127.0.0.1	local.dsde-dev.broadinstitute.org` is in your `/etc/hosts` file.
5. Run `./config/docker-rysnc-local-orch.sh`
6. Go to https://local.dsde-dev.broadinstitute.org:10443/.  You should see orch's swagger page, be able to auth, and try out the APIs
